src/jsonrpc/miner_jsonrpc_ledger.erl

```diff
@@ -157,5 +157,33 @@ last_challenge(_Height, undefined) -> <<"undefined">>;
 last_challenge(Height, LC) -> Height - LC.
 
 format_ledger_validator(Addr, Val, Ledger, Height, Verbose) ->
-    InfoPairs = blockchain_ledger_validator_v1:print(Val, Height, Verbose, Ledger),
-    maps:from_list(InfoPairs ++ [ {name, ?BIN_TO_ANIMAL(Addr)}, {address, ?BIN_TO_B58(Addr)} ]).
+    Penalties = blockchain_ledger_validator_v1:calculate_penalties(Val, Ledger),
+    OwnerAddress = blockchain_ledger_validator_v1:owner_address(Val),
+    LastHeartbeat = blockchain_ledger_validator_v1:last_heartbeat(Val),
+    Stake = blockchain_ledger_validator_v1:stake(Val),
+    Status = blockchain_ledger_validator_v1:status(Val),
+    Version = blockchain_ledger_validator_v1:version(Val),
+    Tenure = maps:get(tenure, Penalties, 0.0),
+    DKG = maps:get(dkg, Penalties, 0.0),
+    Perf = maps:get(performance, Penalties, 0.0),
+    TotalPenalty = Tenure+Perf+DKG,
+    VerboseItems = case Verbose of
```

Comment:
Can you just remove the Verbose option altogether (i..e remove it as an argument to this function too) and just return these items too?

Reply:
Absolutely. I didn't like the verbose option anyway.